dumpling/export/dump.go

```diff
@@ -298,6 +298,27 @@ func (d *Dumper) startWriters(tctx *tcontext.Context, wg *errgroup.Group, taskCh
 func (d *Dumper) dumpDatabases(tctx *tcontext.Context, metaConn *sql.Conn, taskChan chan<- Task) error {
 	conf := d.conf
 	allTables := conf.Tables
+
+	// policy should be created before database
+	if conf.ServerInfo.ServerType == version.ServerTypeTiDB {
+		policyNames, err := ListAllPlacementPolicyNames(metaConn)
```

Comment:
this should be skipped if the `ServerVersion` is too low.

Reply:
yes, but I'm not sure which version is suitable? Do we plan cherry-pick related pr like https://github.com/pingcap/tidb/pull/27478 to v5.3.0? @morgo

Reply:
@sylzd maybe instead of version checking we could do feature checking, and gracefully ignoring the "Unknown table" from `ListAllPlacementPolicyNames()`.

Reply:
LGTM~